google/resource_compute_backend_service.go

```diff
@@ -203,6 +229,7 @@ func resourceComputeBackendServiceRead(d *schema.ResourceData, meta interface{})
 	d.Set("self_link", service.SelfLink)
 	d.Set("backend", flattenBackends(service.Backends))
 	d.Set("connection_draining_timeout_sec", service.ConnectionDraining.DrainingTimeoutSec)
+	d.Set("iap", flattenIap(service.Iap))
```

Comment:
will service.Iap ever be nil?

Reply:
Only in-so-far as the service object doesn't initialize it a few lines up from here. I'll add a check in the flatten function.